app/src/pages/inside/settingsPage/settingTabs/demoDataTab/demoDataTabForm/demoDataTabForm.scss

```diff
@@ -0,0 +1,46 @@
+.input-wrapper {
+  width: 100%;
+  height: 30px;
+  max-width: 320px;
+  margin-bottom: 25px;
+  box-sizing: border-box;
+  @media (max-width: $SCREEN_XS_MAX) {
+    display: none;
+  }
+}
+
+.generate-button {
+  display: block;
+  width: 155px;
+  height: 30px;
+  margin-bottom: 15px;
+  box-sizing: border-box;
+  @media (max-width: $SCREEN_XS_MAX) {
+    margin: 0 auto;
+  }
+}
+
+.generate-button-title {
+  width: 129px;
+  height: 18px;
+  font-size: 13px;
+}
+
+.preloader-wrapper {
+  display: flex;
+  margin-bottom: 35px;
+}
+
+.preloader-icon {
+  width: 50px;
```

Comment:
Seems like it's not needed. Preloader is already 50px width

Reply:
Fixed